TRANSLATING.md

```diff
@@ -173,6 +187,13 @@ i18nReady: true
 // Rest of the file's content is here...
 ```
 
+
+### Code Samples
+
+We have lots of code samples throughout our docs, and although we **recommend translating comments**, as they give a contextual clue of what's happening in the code, each language is **free to decide** if they want or not to translate titles, variables, string values, function names, etc.
+
+Be wary that if code samples are being translated, you may need to update some of the code sample's highlighted lines. Read the [Code Samples section](https://github.com/withastro/docs/blob/main/WRITING.md#code-samples) in our Writing Guide to know more about our syntax.
```

Comment:
I think “aware” might be better here; “wary” implies more “be fearful” (which depends on how you feel about Hippo’s code highlighting syntax 😆)

```suggestion
Be aware that if code samples are being translated, you may need to update some of the code sample's highlighted lines. Read the [Code Samples section](https://github.com/withastro/docs/blob/main/WRITING.md#code-samples) in our Writing Guide to know more about our syntax.
```